bukkit/src/main/java/com/leonardobishop/quests/bukkit/tasktype/type/PlaytimeTaskType.java

```diff
@@ -40,6 +41,8 @@ public PlaytimeTaskType(BukkitQuestsPlugin plugin) {
 
     @Override
     public void onReady() {
+        boolean ignoreAFK = plugin.getQuestsConfig().getBoolean("options.playtime-ignores-afk", false);
```

Comment:
This is probably better suited as a task configuration key (similar with CoreProtect in the mining tasks), rather than a config.yml key